code/playerman/playercontrol.cpp

```diff
@@ -1379,7 +1379,15 @@ void player_level_init()
 	Viewer_external_info.preferred_distance = 0.0f;
 	Viewer_external_info.current_distance = 0.0f;
 
-	Viewer_mode = 0;
+	
+	if (Chase_view_default == true)
```

Comment:
This is a redundant boolean comparison, which some compilers will complain about.